src/containers/home/home.component.jsx

```diff
@@ -1,49 +1,54 @@
-import React from "react";
-
+import React, { useEffect, useState } from "react";
 import { auth } from "database";
-
-import { fetchPosts } from "database/posts";
-import { addPost } from "database/addPost";
-
 import { useSelector } from "react-redux";
+import CreatePost from "containers/create-post/create-post.component";
+import Post from "containers/post/post.component";
 
 const Home = (props) => {
+  const [posts, setPosts] = useState([]);
   const userFromState = useSelector((state) => state.users.user);
 
   const signOut = () => {
     auth.signOut();
     props.logoutThunk();
   };
-
+  const postsFromState = useSelector((state) => state.posts);
+
+  useEffect(() => {
+    props.getAllPostsToRenderThunk();
+    setPosts(postsFromState);
+  }, [posts]);
+
+  let displayPost;
+
+  if (postsFromState.posts.length > 0) {
+    displayPost = postsFromState.posts.map((post) => {
+      return (
+        <Post
+          body={post.body}
+          option_a={post.option_a}
```

Comment:
Javascript frontend variables should be in camelCase such as `optionA` as opposed to database variables should be in camel case such as `option_a`

Reply:
done